### PR TITLE
Enable Travis for CI

### PR DESCRIPTION
**Pull request by jvaltin**
This branch enables Travis for continuous integration. It uses a Docker container which may also be used locally to test building of the Android package.

### CODE REVIEW

**Comment by stephaniestroka** [comment]
LGTM, thanks so much! :-)

File: Dockerfile.travis [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+RUN npm install -g cordova
+RUN npm install -g buffer-reverse
+RUN npm install -g browserify
+
+RUN cordova prepare
+RUN npm install
+RUN npm install bitcore-lib
+
+RUN cordova platform add android
```
Comment (douglasbakkum) [inline Dockerfile.travis]: Add iOS?
I believe travis has OSX instances such that it may be possible to check the iOS compilation too.

**Comment by douglasbakkum** [comment]
LGTM too

Add an iOS build? I believe travis has OSX instances such that this may be possible.

**Comment by jvaltin** [comment]
I've now added a MacOS build process as part of Travis and it produces an iOS application as expected.

**Comment by jvaltin** [comment]
This is ready for merge.

**Pull request merged**